pkg/kubelet/util/swap/swap_util.go

```diff
@@ -46,6 +47,13 @@ func IsTmpfsNoswapOptionSupported(mounter mount.Interface, mountPath string) boo
 			return false
 		}
 
+		if inuserns.RunningInUserNS() {
+			// Turning off swap in unprivileged tmpfs mounts unsupported
+			// https://github.com/torvalds/linux/blob/v6.8/mm/shmem.c#L4004-L4011
+			// https://github.com/kubernetes/kubernetes/issues/125137
+			return false
```

Comment:
Would you mind adding a log message here, so it would be clear why tmpfs noswap is not being used?
Perhaps something like:
```go
klog.InfoS("Running under a user namespace - tmpfs noswap is not supported")
```

Reply:
done